src/bridges-deep-dive-dodao-academy.yaml

```diff
@@ -90,18 +91,21 @@ steps:
     name: Evaluation
     stepItems:
       - answerKeys:
+          - f85a6ecc
           - 6db3906f
+          - 3975ac56
+          - bdf62e57
         choices:
-          - content: A
+          - content: Sequencers
             key: f85a6ecc
-          - content: B
+          - content: Challengers
             key: 6db3906f
-          - content: C
+          - content: Executors
             key: 3975ac56
-          - content: D
+          - content: Validators
             key: bdf62e57
-        content: Q1
-        type: SingleChoice
+        content:  What type of workers are involved in assessing the security of a validating bridge?
```

Comment:
are `Sequencers` and `Executors` responsible for validation?

Reply:
Sir, the question is not about validation, it's about assessing the security required for validation